modules/web/js/ballerina/visitors/sizing-utils.js

```diff
@@ -74,6 +74,14 @@ class SizingUtil {
         };
     }
 
+    /**
+     * Returns the width of a given string when rendered as svg text according to given options
+     * Unlike `getTextWidth` this method does not try to to truncate the given text depending on its length.
+     * @param {string} text - The string of which the length is measured
+     * @param {Object} options
+     * @param {number} options.fontSize
```

Comment:
Sure. Updated the PR. https://github.com/ballerinalang/composer/pull/1971/files#diff-f5fa26da69f021f31580fdde94ce23c5R81